audisp-tacplus.c

```diff
@@ -604,14 +500,14 @@ static void get_acct_record(auparse_state_t *au, int type)
     /*
      * loguser is always set, we bail if not.  For ANOM_ABEND, tty may be
      *  unknown, and in some cases, host may be not be set.
-     */
-    send_tacacs_acct(loguser, tty?tty:"UNK", host?host:"UNK", logbase, acct_type, taskno);
-
-    if(host)
-        free(host);
+     */
+    char* fixed_log_buffer = remove_user_secret(logbase);
```

Comment:
>remove_user_secret [](http://example.com/codeflow?start=29&length=18)

Sorry for a late question, why not replace secret in logbase directly? I notice you malloc and free a buffer.

Reply:
Fixed